rfcs/0146-meta-categories.md

```diff
@@ -249,40 +278,42 @@ The most immediate drawbacks are:
 
 As said above, categorization is very traditional among software collections. It
 is not hard to cite examples in this arena; the most interesting ones I have
-found are listed above (linked at [references section](#references))
+found are listed below (linked at [references section](#references)):
 
 - FreeBSD Ports;
 - Debtags;
+- Appstream Project;
 
 # Unresolved questions
 [unresolved]: #unresolved-questions
 
-Still unsolved is what data structure is better suited to represent a category.
+There are some still unsolved issues:
+
+- What data structure is suitable to represent a category?
 
-- For now we stick to a set `{ name, description }`.
+  - For now we stick to the most natural: a set `{ name, description }`.
+
+- Should we have a set of primary, "most important" categories with mandatory
+  status, in the sense each package should set at least one of them?
+  - The answer is most certainly positive.
 
 # Future work
 [future]: #future-work
 
-Given the typical complexities that arise from categorization, and expecting
-that regular maintainers are not expected to understand its minuteness, it is
-strongly recommended the creation of a team entrusted to manage issues related
-to categorization, including but not limited to:
-
-- Update documentation.
-- Curate the categories.
-- Update Continuous Integration.
-- Integrate categorization over the current codebase.
-
-Such a team should receive authority to carry out their duties:
-
-- Coordinaton of efforts to import, integrate and update categorization of
-  packages.
-- Disputations over categorization, especially corner cases.
-- Decisions about when a Continuous Integration check for categorisation is
-  ready to be developed with gray/neutral failure statuses, and when a CI check
-  with a good track record in gray mode can be upgraded to red/blocking
-  failures.
+- Create the [categorization team](#categorization-team)
+- Carry out the duties correlated to categorization, including but not limited
+  to:
+
+  - Decide between possibilities of implementation;
+  - Documentation updates;
+  - Category curation, integration and updates;
+  - Continuous Integration updates and adaptations;
+  - Coordinaton of efforts to import, integrate and update categorization of
+    packages;
+  - Litigations and disputations:
+    - Solve disputations over categorization, especially corner cases;
+    - Solve and enforce implementation issues from CI checks
+      - When should a CI check about categorization be treated as blocking?
```

Comment:
```suggestion
      - When should a CI check about categorization be converted to blocking?
```
(to signal that adding blocking checks directly is rarely a good idea)

Reply:
OK.